cmd/cli/support_bugreport.go

```diff
@@ -14,6 +14,8 @@ import (
 
 	"github.com/openservicemesh/osm/pkg/bugreport"
 	"github.com/openservicemesh/osm/pkg/constants"
+
+	policyClientset "github.com/openservicemesh/osm/pkg/gen/client/policy/clientset/versioned"
```

Comment:
Generally speaking about our repo, not so much about this PR - `policyClientset` sounds so broad and generic. I wonder if at some point we should prefix the OSM clientsets with `osm` so when reading the code it is clear that this is some OSM specific CRD. So this would be `osmPolicyClientset`.